host/4/bullseye/amd64/powershell/powershell7.Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0.0-preview.7-bullseye-slim
+FROM mcr.microsoft.com/dotnet/aspnet:6.0.0-rc.1
```

Comment:
do these need to be `bullseye-slim`?

Reply:
@brettsam  I believe both the `6.0-bullseye-slim` and ` 6.0.0-rc.1` tags points to same image. I chose rc1 to be consistent with other instances where we used `rc.1` . Do you think we should use  `6.0-bullseye-slim` tag instead ?

https://hub.docker.com/_/microsoft-dotnet-aspnet

Reply:
yeah. 6.0-bullseye-slim is much more descriptive :)

Reply:
and i assume even if a rc2 image gets published, we dont have to update anything here

Reply:
Let me push another iteration to switch to `bullseye-slim` so it is consistent with previous iterations of v4 work.

Reply:
Switched back to `6.0.0-rc.1-bullseye-slim`.